src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -1076,6 +1081,33 @@ def extract_assay_expression(parser_obj):
     path_validator = PathValidator(input_dict=parser_dict, project=project, entity_describe=entity_describe, error_handler=err_exit)
     path_validator.validate(check_list_assays_invalid_combination=True)
 
+    # # Validating input JSON
+    # if parser_obj.input_json:
```

Comment:
This is not a parser obj (name could indicate ArgumentParser) but a Namespace object which is being refferred to as `args` across dx-toolkit

Reply:
Great catch. Thank you! I'll take a note of this and will modify it later. There are many open PRs at the moment, so I don't want to disrupt other devs' work.